Codecademy/gamut
# feat(Modal): adding aria-live as a prop to ModalContainer

### Overview

As a part of this a11y ticket I was asked to turn off a modal's aria-live: [A11Y-579](https://codecademy.atlassian.net/browse/A11Y-579), but I couldn't because it was hardcoded to "polite" in gamut. Here, I'm adding the ability to change the aria-live but also keeping "polite" as a default





### PR Checklist

- [x] Related to JIRA ticket: [A11Y-579](https://codecademy.atlassian.net/browse/A11Y-579)

#### Testing Instructions

1. go to the preview LE below
2. View a solution diff modal, see that it's aria-live is off in the help text
3. Go anywhere else where there is a modal and see that it still has aria-live="polite"



[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/36196)  | [Monolith Env]( ) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/2883)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-2883)   |
| LE link | [Same as Portal link above, but LE](https://github.com/codecademy-engineering/mono/pull/2883) | [LE Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-2883)  |




[A11Y-579]: https://codecademy.atlassian.net/browse/A11Y-579?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[A11Y-579]: https://codecademy.atlassian.net/browse/A11Y-579?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
